automation/Dockerfile

```diff
@@ -16,7 +16,10 @@ RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == \
 RUN yum -y install NetworkManager epel-release git cairo* && \
     yum -y install python2-pip gcc python-devel gobject-introspection-devel && \
     yum clean all && \
-    pip install -U pip setuptools pbr tox
+    pip install -U pip setuptools pbr tox && \
+    \
+    sed 's/#level=TRACE/level=TRACE/' /etc/NetworkManager/NetworkManager.conf > /etc/NetworkManager/NetworkManager.conf && \
```

Comment:
seems `sed -i` would be more robust. 

but anyway, it seems nicer to leave the main configuration file unchanged:
```
  echo -e "[logging]\nlevel=TRACE\ndomains=ALL\n" > /etc/NetworkManager/conf.d/97-docker-build.conf
```